### ISSUE TITLE
Unable to leave TARDIS after Real World Flight

### ISSUE THREAD

**Issue by BadDevilGrind** [Spectre0987/TardisMod#265]
In 0.1.0C, after using the Real World Flight system, whenever you try to leave the TARDIS after it has landed you are transported back to the interior, and further attempts to leave through the exterior door results in you simply physically walking through the door.

![2019-04-24_12 11 23](https://user-images.githubusercontent.com/1296967/56652006-956d5080-668a-11e9-9da2-b102c7a59afe.png)

Additionally, removing the Dimensional Dams results in that you are ejected from the TARDIS, but are unable to move. Killing yourself with /kill resets this

I also noticed, that after using the Real World Flight system after being transported back to the interior, it seems to create a duplicate of the TARDIS in the real world, which cannot be opened with a key.

![2019-04-24_12 11 31](https://user-images.githubusercontent.com/1296967/56652016-9a320480-668a-11e9-8281-0bda509c467d.png)

Addendum: After quitting the world and loggin back in (singleplayer), the key can be used to open any of the TARDIS clones, and they all go to the same interior, but when exiting you always end up outside the "real" one.

![2019-04-24_12 10 47](https://user-images.githubusercontent.com/1296967/56652024-9d2cf500-668a-11e9-9653-1bcf77a99713.png)

**Comment by BadDevilGrind** [comment]
Update:

I replicated the issue of not being able to leave again, but when I run the /forge setdim cmd, I am not transported to the dimension, but instead to the center of the console room of the TARDIS. The only exception I found was teleporting to the End, but teleporting to the Overworld or the Nether results in being teleported to the console room.

This only happends after using the Real World Flight system once and landing and exiting the RWF system.
